### DISCUSSION TITLE
A new home: iok.dev

### DISCUSSION THREAD

**Discussion by bradleyjkemp** [phish-report/IOK#241] [Announcements]
There's a new place to try out IOK rules: [iok.dev](https://iok.dev) 🎉 

Previously we had a little editor at the bottom of [phish.report/IOK](https://phish.report/IOK) but we've upgraded this to it's own full-page experience on iok.dev

Going forward this is where docs, etc. for the project will live

**Comment by IlluminatiFish** [comment]
🎉

**Comment by LightningDev23** [comment]
Nice, thank you!
